src/streams/StreamItem.js

```diff
@@ -151,13 +160,23 @@ export default function StreamItem(props: Props): Node {
       }}
     >
       <View style={wrapperStyle}>
+        {isCollapsible === true && (
+          <Touchable onPress={handleCollapse && (() => handleCollapse(streamId))}>
+            {isUnCollapsed === true ? (
+              <IconUnCollapse style={componentStyles.collapseIcon} size={20} color={color} />
```

Comment:
OTOH a handy thing about that visual feedback -- handy in development, that is -- is that it makes it easy to see from a screencap how far the touchable area extends!

In general, we aim for touch targets of at least 48x48. See discussion here:
https://material.io/design/usability/accessibility.html#layout-and-typography

(For background on the various units for sizes, see here: https://github.com/zulip/zulip-mobile/blob/main/docs/background/layout.md
In short, 1 dp = 1 px = "1" as seen in RN layout props.)

These stream-item rows are less than 48px tall, so the touchable area should extend the full height of the row. When it's smaller, it's often difficult for people to hit.

Ideally it would also be 48px wide, even though it's less than 48px tall.

Reply:
- Here is what the `StreamItem` looks like with the width of the `Touchable` at 48.

<img width="324" alt="Screen Shot 2022-09-04 at 9 54 02 AM" src="https://user-images.githubusercontent.com/50177345/188324744-938cf6c5-a525-4097-866f-013b5746d1c9.png">

- Currently the surrounding `View` has padding added to it from `listItem` in `miscStyles.js`. With that padding there, if the user clicks in the padding area that is just outside the `Pressable` it will take them to the individual Stream instead of collapse/expand. Should I approach a way to conditionally remove this padding in the area where the `Pressable` is if the StreamItem is one that will be collapsible? 
- If so this should also bring the `Caret` 16px to the left which may make it look a little less wide.

Reply:
(Replied [in the main thread](https://github.com/zulip/zulip-mobile/pull/5467#issuecomment-1238712214) below.)